### PR TITLE
LOG-3813: fix cloudwatch,gcl tls security profiles

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* fixes tls cipher and min version for cloudwatch and gcl

### Links
https://issues.redhat.com/browse/LOG-3813

### CODE REVIEW

File: internal/generator/vector/output/security/security.go [hunk 1]
```diff
@@ -60,24 +60,27 @@ func NewTLSConf(o logging.OutputSpec, op generator.Options) TLSConf {
 }
 
 func addTLSSettings(o logging.OutputSpec, secret *corev1.Secret, conf *TLSConf) (TLSConf, bool) {
```
Comment (syedriko) [inline internal/generator/vector/output/security/security.go:62]: One of those TLSConf's is too many. The signature of addTLSSettings should be
`(o logging.OutputSpec, secret *corev1.Secret, conf *TLSConf)  bool`
or
`(o logging.OutputSpec, secret *corev1.Secret) *TLSConf`

File: internal/generator/vector/output/cloudwatch/output_cloudwatch_test.go [hunk 2]
```diff
@@ -438,6 +459,37 @@ tls.verify_certificate = false
...
 		})
 
+		It("should generate an output.tls config block", func() {
+			output.Cloudwatch.GroupBy = ""
+			secrets[vectorTLSSecret].Data = map[string][]byte{
+				"aws_access_key_id":     []byte(keyId),
+				"aws_secret_access_key": []byte(keySecret),
+			}
+			expConf := `
+	` + transformBegin + `
```
Comment (syedriko) [inline internal/generator/vector/output/cloudwatch/output_cloudwatch_test.go:469]: indentation is off here

File: internal/generator/vector/output/gcl/gcl_test.go [hunk 3]
```diff
@@ -125,6 +125,60 @@ verify_hostname = false
...
+
+[sinks.gcl_tls.resource]
+type = "k8s_node"
+node_name = "{{hostname}}"
+
+[sinks.gcl_tls.tls]
+enabled = true
+min_tls_version = "` + defaultTLS + `"
+ciphersuites = "` + defaultCiphers + `"
+verify_certificate = false
```
Comment (syedriko) [inline internal/generator/vector/output/gcl/gcl_test.go:180]: Why don't we go for the full complement of TLS settings here: CA, cert, key and passphrase?

**Comment by jcantrill** [comment]
/test e2e-target
--


<br class="Apple-interchange-newline">

**Pull request merged**